### PR TITLE
[keymap] curly quotes

### PR DESCRIPTION
**Pull request by callum-oakley**
Just upstreaming some keymap changes. :)

### CODE REVIEW

File: keyboards/planck/keymaps/callum/rules.mk [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #   the appropriate keymap folder that will get included automatically
```
Comment (fauxpark) [inline keyboards/planck/keymaps/callum/rules.mk:3]: Just while you're here, this entire file can be reduced to:
```make
BOOTMAGIC_ENABLE = no
MOUSEKEY_ENABLE = no
CONSOLE_ENABLE = no
COMMAND_ENABLE = yes
MIDI_ENABLE = no
AUDIO_ENABLE = yes
RGBLIGHT_ENABLE = no
```

Comment (callum-oakley) [reply keyboards/planck/keymaps/callum/rules.mk]: good to know, thanks!

File: keyboards/planck/keymaps/callum/readme.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ This is a layout for the grid planck, built with a few ideals in mind:
 
 - Symbols should be arranged so that the most frequently used are easiest to
   reach. This includes numbers, and lower numbers are more commonly used than
-  higher ones. (number arrangement borrowed from [dustypomeleau's minidox
+  higher ones. (number arrangement borrowed from [dustypomeleau’s minidox
   layout][].
```
Comment (Erovia) [inline keyboards/planck/keymaps/callum/readme.md:27]:
```suggestion
  layout][]).
```

Comment (callum-oakley) [reply keyboards/planck/keymaps/callum/readme.md:27]: Thanks :)

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**